mirrord-layer/src/lib.rs

```diff
@@ -498,6 +500,14 @@ fn enable_hooks(enabled_file_ops: bool, enabled_remote_dns: bool) {
 
     unsafe {
         let _ = replace!(&mut interceptor, "close", close_detour, FnClose, FN_CLOSE);
+        #[cfg(target_os = "macos")]
```

Comment:
I saw somewhere that it exists for other platforms as well. Just make it fallible.